src/test/java/plugins/LdapPluginTest.java

```diff
@@ -205,7 +249,7 @@ public void login_use_fallback_server() {
         int freePort = this.findAvailablePort();
         LdapDetails ldapDetails = new LdapDetails("", 0, ldapContainer.getManagerDn(), ldapContainer.getManagerPassword(), ldapContainer.getRootDn());
         // Fallback-Config: primary server is not running, alternative server is running docker fixture
-        ldapDetails.setHostWithPort("localhost:" + freePort + " localhost:" + ldapContainer.getPort());
+        ldapDetails.setHostWithPort(ldapContainer.getHost()+":" + freePort + " " + ldapContainer.getHost() + ":" + ldapContainer.getPort());
```

Comment:
Will change it. Just want to point out that this issue will still be present for localhost and linux. Because `ldapContainer.getHost() == localhost` in that case..